### ISSUE TITLE
Create an endpoint that returns the running version of Aquarius

### ISSUE THREAD

**Issue by ttmc** [oceanprotocol/aquarius#97]
@MarcusJones said he connected to a remote instance of Aquarius and wasn't sure what version of Aquarius was running at the other end.

**Proposal:** Return the software name and version in the response when you visit the root/docroot/server endpoint. For example, if you make a GET request to:

https://node532.aquarius-hosts.net:5000/

with no `api/v1/...` on the end, the HTTP response might look something like:

```json
{
  "software": "Aquarius",
  "version": "3.14.1"
}
```

More information could be included in the response but this would be a good starting point.

**Comment by mikera** [comment]
Should we perhaps generalise this to an Agent Identity API that all Ocean agents can provide as a standard?

I can see it being useful for discovery and identification of all kinds of agents, e.g.

```
{
  "software": "ReferenceMarketplace",
  "version": "1.3.0",
  "operatedBy" : "ACME Data Company"
}
```

**Comment by ttmc** [comment]
I don't want to balloon this issue into a big one. A _lot_ could be added to the response to the docroot endpoint. Requests for additional things should maybe be done using new issues, ideally from users who have specific problems that need solving.

**Comment by ttmc** [comment]
Resolved by PR #98
